src/arviz_plots/plots/evolutionplot.py

```diff
@@ -0,0 +1,449 @@
+    group="posterior",
+    coords=None,
+    sample_dims=None,
+    # plot specific arguments
+    relative=False,
+    rug=False,
+    rug_kind="diverging",
+    n_points=20,
+    # extra_methods=False,
+    # min_ess=400,
+    # more base arguments
+    plot_collection=None,
+    backend=None,
+    labeller=None,
+    aes_map=None,
+    plot_kwargs=None,
+    stats_kwargs=None,
+    pc_kwargs=None,
+):
+    """Plot estimated effective sample size plots for increasing number of iterations.
+
+    Parameters
+    ----------
+    dt : DataTree or dict of {str : DataTree}
+        Input data. In case of dictionary input, the keys are taken to be model names.
+        In such cases, a dimension "model" is generated and can be used to map to aesthetics.
+    var_names : str or sequence of str, optional
+        One or more variables to be plotted.
+        Prefix the variables by ~ when you want to exclude them from the plot.
+    filter_vars : {None, “like”, “regex”}, default None
+        If None, interpret `var_names` as the real variables names.
+        If “like”, interpret `var_names` as substrings of the real variables names.
+        If “regex”, interpret `var_names` as regular expressions on the real variables names.
+    group : str, default "posterior"
+        Group to be plotted.
+    coords : dict, optional
+    sample_dims : str or sequence of hashable, optional
+        Dimensions to reduce unless mapped to an aesthetic.
+        Defaults to ``rcParams["data.sample_dims"]``
+    relative : bool, default False
+        Show relative ess in plot ``ress = ess / N``.
+    rug : bool, default False
+        Add a `rug plot <https://en.wikipedia.org/wiki/Rug_plot>`_ for a specific subset of values.
+    rug_kind : str, default "diverging"
+        Variable in sample stats to use as rug mask. Must be a boolean variable.
+    n_points : int, default 20
+        Number of subsets in the evolution plot.
+    extra_methods : bool, default False
+        Plot mean and sd ESS as horizontal lines.
+    min_ess : int, default 400
+        Minimum number of ESS desired. If ``relative=True`` the line is plotted at
+        ``min_ess / n_samples`` as a curve following the ``min_ess / n`` dependency
+    plot_collection : PlotCollection, optional
+    backend : {"matplotlib", "bokeh"}, optional
+    labeller : labeller, optional
+    aes_map : mapping of {str : sequence of str or False}, optional
+        Mapping of artists to aesthetics that should use their mapping in `plot_collection`
+        when plotted. Valid keys are the same as for `plot_kwargs`.
+    plot_kwargs : mapping of {str : mapping or False}, optional
+        Valid keys are:
+        * "bulk" -> passed to :func:`~arviz_plots.visuals.scatter_xy` and
+        :func:`~arviz_plots.visuals.line_xy`
+        * "tail" -> passed to :func:`~arviz_plots.visuals.scatter_xy` and
+        :func:`~arviz_plots.visuals.line_xy`
+        * divergence -> passed to :func:`~.visuals.trace_rug`
```

Comment:
This key should be "rug" instead. The most common thing to use (and therefore default for `rug_kind` are divergences) but they are not the only option. As we agreed on smaller set of fixed keys in #66, I think this key should be "rug" instead of `rug_kind`.

Note this will require multiple changes in the function code